test/integration/auth.spec.ts

```diff
@@ -24,13 +23,16 @@ import firebase from '@firebase/app';
 import '@firebase/auth';
 import {clone} from 'lodash';
 import {
-  generateRandomString, projectId, apiKey, noServiceAccountApp, cmdArgs,
+  generateRandomString, noServiceAccountApp, cmdArgs, apiKey, projectId
 } from './setup';
 import url = require('url');
 import * as mocks from '../resources/mocks';
 import { AuthProviderConfig } from '../../src/auth/auth-config';
 import { deepExtend, deepCopy } from '../../src/utils/deep-copy';
 import { User, FirebaseAuth } from '@firebase/auth-types';
+import { TenantOptions, SecondFactor, UserImportOptions, PhoneMultiFactorInfo, UserImportRecord, UserRecord, TenantAwareAuth, auth } from '../../src/auth/';
+import { defaultApp } from './setup';
```

Comment:
Nit: Move this import to the import on line 27

Reply:
As-below, this isn't needed anymore now that `auth` top-level service function accepts an optional `app` which defaults to `defaultApp` anyway